File: proyecto_admin/app/Models/Semilla.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\BelongsTo;

class Semilla extends Model
{
    protected $table = 'semillas';
    public $timestamps = false;

    protected $fillable = [
        'idUsuario',
        'idPrototipo',
        'uid',
        'color',
        'tamano',
        'peso',
        'estado',
        'fechaRegistro'
    ];

    public function usuario(): BelongsTo
    {
        return $this->belongsTo(Usuario::class, 'idUsuario');
    }
}

File: app_movil/app/src/main/java/com/example/appfrijol/domain/model/Semilla.kt
package com.example.appfrijol.domain.model

data class Semilla(
    val id: Int,
    val color: String,
    val tamano: String,
    val peso: Double,
    val estado: String,
    val fechaRegistro: String,
    val idUsuario: Int,
    val idPrototipo: Int
)